<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>564</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>843</width>
    <height>564</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>843</width>
    <height>564</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>APFEL GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>APFEL Results:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Quarks and Gluon PDFs</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QGraphicsView" name="graphicsView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>467</width>
               <height>400</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>467</width>
               <height>400</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Photon PDF</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QGraphicsView" name="graphicsView_2"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Open Calculator</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Save plot</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>APFEL Configuration</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="11" column="0" colspan="2">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Generate LHgrid</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLineEdit" name="lineEdit_5">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Flavors: (FFNS)</string>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLineEdit" name="lineEdit_4">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="17" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>u_v sum rule =</string>
           </property>
          </widget>
         </item>
         <item row="17" column="1">
          <widget class="QLineEdit" name="lineEdit_6">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="comboBox_4">
           <item>
            <property name="text">
             <string>POLE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MSbar</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>100.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Output LHgrid:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0" colspan="2">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Compute</string>
           </property>
          </widget>
         </item>
         <item row="19" column="1">
          <widget class="QLineEdit" name="lineEdit_7">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Input LHgrid:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Final scale (GeV):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000000.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>QCD</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QED</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QECDP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QCEDP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QavDP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QECDS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QCEDS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>QavDS</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="spinBox">
           <property name="minimum">
            <number>3</number>
           </property>
           <property name="maximum">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="19" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>d_v sum rule =</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Initial scale (GeV):</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>alphaQED(mu2F)=</string>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Mom. sum rule =</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Theory:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Perturbative Order:</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>HQ scheme:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Scheme:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>alphaQCD(mu2F) =</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBox_3">
           <item>
            <property name="text">
             <string>VFNS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FFNS</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBox_2">
           <item>
            <property name="text">
             <string>NLO (LO QED)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>NNLO (LO QED)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LO (LO QED)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="20" column="1">
          <widget class="QLineEdit" name="lineEdit_8">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="20" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>s_v sum rule =</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>label_2</zorder>
      <zorder>doubleSpinBox</zorder>
      <zorder>label</zorder>
      <zorder>doubleSpinBox_2</zorder>
      <zorder>groupBox_2</zorder>
     </widget>
    </item>
   </layout>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
